src/Main.java

```diff
@@ -35,33 +35,52 @@ public static void main(String[] args) {
         short totalSheets = 480;
         System.out.println("На каждого ученика рассчитано " + totalSheets / (studentsOfTheFirstTeacher + studentsOfTheSecondTeacher + studentsOfTheThirdTeacher) + " листов бумаги");
 
-        byte equipmentPerformancePerMinute = 8;
-        System.out.println("За 20 минут машина произвела " + equipmentPerformancePerMinute * 20 + " штук бутылок");
-        System.out.println("За сутки машина произвела " + equipmentPerformancePerMinute * 1440 + " штук бутылок");
-        System.out.println("За три дня машина произвела " + equipmentPerformancePerMinute * 4320 + " штук бутылок");
-        System.out.println("За один месяц машина произвела " + equipmentPerformancePerMinute * 44640 + " штук бутылок");
+        byte equipmentPerformanceIn2Minutes = 16;
+        short equipmentPerformanceIn20Minutes = (short) (equipmentPerformanceIn2Minutes * 10);
```

Comment:
```suggestion
        short equipmentPerformanceIn20Minutes = (short) (equipmentPerformanceIn1Minute * 20);
```
в уме ничего не считаем